ADAL/src/request/ADAcquireTokenSilentHandler.m

```diff
@@ -126,6 +127,14 @@ - (void)acquireTokenByRefreshToken:(NSString*)refreshToken
         request_data[OAUTH2_SCOPE] = _requestParams.scope;
     }
     
+    if (![ADHelpers isADFSInstance:_requestParams.authority])
+    {
+        NSString* enrollId = [ADEnrollmentGateway enrollmentIDForUniqueAccountID:nil userID:(cacheItem.userInformation.userId ? cacheItem.userInformation.userId : _requestParams.identifier.userId) error:NULL];
```

Comment:
why are we ignoring the error?

Reply:
Although we are not going to stop the whole acquire token flow because of this error, I think we could log it here. It will be helpful diagnosis purpose :)

Reply:
This method originally only returned the error returned by NSJSONSerialization, and logged separately.  It's since been updated to accept ADAuthenticationErrors, and so passing the error here should trigger logging automatically.